zebrad/src/commands/start.rs

```diff
@@ -328,49 +393,133 @@ impl StartCmd {
+
+                    warn!(
+                        %sync_percent,
+                        ?current_height,
                         ?remaining_sync_blocks,
-                        "initial sync might have stalled, or estimated tip is wrong. \
-                         Hint: check that your computer's clock and time zone are correct"
+                        ?after_checkpoint_height,
+                        %time_since_last_state_block,
+                        "initial sync is very slow, and state is below the highest checkpoint. \
+                         Hint: check your network connection, \
+                         and your computer clock and time zone. \
+                         Dev Hint: were the checkpoints updated in the last {} minutes?",
+                        min_minutes_after_checkpoint_update,
                     );
-                } else if is_close_to_tip {
+                } else if is_syncer_stopped {
                     // We've stayed near the tip for a while, and we've stopped syncing lots of blocks.
-                    //
-                    // TODO: reduce this log level after full sync tests are merged,
-                    //       and have been stable for a while (#1592)
+                    // So we're mostly using gossiped blocks now.
                     info!(
                         %sync_percent,
+                        ?current_height,
                         ?remaining_sync_blocks,
-                        "finished initial sync to chain tip"
+                        %time_since_last_state_block,
+                        "finished initial sync to chain tip, using gossiped blocks",
                     );
                 } else if remaining_sync_blocks <= MAX_CLOSE_TO_TIP_BLOCKS {
                     // We estimate we're near the tip, but we have been syncing lots of blocks recently.
+                    // We might also be using some gossiped blocks.
                     info!(
                         %sync_percent,
+                        ?current_height,
                         ?remaining_sync_blocks,
-                        "finished initial sync to chain tip"
+                        %time_since_last_state_block,
+                        "close to finishing initial sync, \
+                         confirming using syncer and gossiped blocks",
                     );
                 } else {
                     // We estimate we're far from the tip, and we've been syncing lots of blocks.
                     info!(
                         %sync_percent,
+                        ?current_height,
                         ?remaining_sync_blocks,
-                        "estimated progress to chain tip"
+                        %time_since_last_state_block,
+                        "estimated progress to chain tip",
+                    );
+                }
+            } else {
+                let sync_percent = format!("{:.frac$}", 0.0f64, frac = SYNC_PERCENT_FRAC_DIGITS,);
```

Comment:
Optional: formatting tweak to become slightly prettier.

```suggestion
                let sync_percent = format!("{:.frac$} %", 0.0f64, frac = SYNC_PERCENT_FRAC_DIGITS,);
```